test/unit/Model.js

```diff
@@ -4717,6 +4717,12 @@ describe("Model", () => {
 
 				expect(result).to.eql("Dynamoose Warning: Passing callback function into transaction method not allowed. Removing callback function from list of arguments.");
 			});
+
+			it("Should not delete keys from object", () => {
```

Comment:
Looks like this is a `transaction.update` call. I think we should add one for just `update` as well.

Reply:
Ah, good point!